packages/nextjs/app/debug/_components/contract/ReadOnlyFunctionForm.tsx

```diff
@@ -32,9 +32,9 @@ export const ReadOnlyFunctionForm = ({
   const [form, setForm] = useState<Record<string, any>>(() =>
     getInitialFormState(abiFunction),
   );
-  const [inputValue, setInputValue] = useState<any>();
+  const [inputValue, setInputValue] = useState<any | undefined>(undefined);
```

Comment:
When the `inputValue` doesnt change, clicking again on `READ` wont show loading. This is not intended as we need to show the current data, the method will not fetch even if a new value is available in the contract. 

Steps :
1. input 1
2. click on READ
3. click on READ ( wont load again )